packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -87,38 +39,23 @@ export default function ScrollFeedFiltersOnboarding({
       }}
     >
       <div className="flex flex-col items-center">
-        {version === ScrollOnboardingVersion.V1 ? (
-          <GaussianBlur version={version} />
-        ) : (
-          <div className="flex relative justify-center items-center mb-10">
-            <GaussianBlur version={version} />
-            <SuperchargeIcon className="w-[4.625rem] h-[4.625rem]" />
-          </div>
```

Comment:
Should this be still rendered? Looking at the condition it renders on v2.

Reply:
Oh totally missed this, it should indeed.
Thought it was a single render at first.